constraints.txt

```
@@ -1 +1,2 @@
 pip<10.0.0
+rpc-zigzag~=1.0.0
```

This pin is too tight.

```suggestion
rpc-zigzag~=1.0
```